core/src/main/scala/scala/pickling/spi/PicklerRegistry.scala

```diff
@@ -56,23 +56,22 @@ trait PicklerRegistry {
     *             In those situations, the pickler should be able to handle arbitrary (existential) type parameters.
     * @param p  The pickler to register.
     */
-  def registerPickler[T](key: String, p: Pickler[T]): Unit
-  def registerPickler[T](p: Pickler[T]): Unit = registerPickler(p.tag.key, p)
+  def registerPickler(key: String, p: Pickler[_]): Unit
+  def registerPickler(p: Pickler[_]): Unit = registerPickler(p.tag.key, p)
```

Comment:
OK, I had the assumption that the existential quantifier wouldn't work with the `registerPickler` since it was expecting a concrete type `T`. I did a quick test in the REPL and it looks like that's not a problem, `Any` would get inferred. But still, I see no reason why we should keep the types there, since we don't use them at all in that part of the API. I completely agree that we should use types to prevent stupid errors to happen, my problem is that I can't imagine any of these errors for this concrete case. Could you please show me what you mean?